### ISSUE TITLE
CLI should not print output when multiple files matched

### ISSUE THREAD

**Issue by fisker** [prettier/prettier#8832]
When formatting files but forgot `--write` flag, the cli will printed all the codes. It's annoying, and not much useful I think.

Suggest only print code, when one `filepath`(not pattern) or `stdin`.

WDYT?

**Comment by brodybits** [comment]
What would be the new behavior? Exit with an error message or just do nothing? (I would favor doing `process.exit(1)` with an error code.)

**Comment by fisker** [comment]
```console
prettier foo.js
```

check file exists
    a) not exist, throw NotFoundError, process.exit(1)
    b) found, keep current behavior

```console
prettier foo.js bar.js
```

throw MultipleInputNotAllowError , process.exit(1)

```console
prettier [foo].js
```

Same as `foo.js`(do not treat as glob pattern)

**Comment by brodybits** [comment]
> throw MultipleInputNotAllowError , process.exit(1)

In that case, I would favor an explanation that check or write option is needed for multiple input files.

> ```
> prettier [foo].js
> ```

I thought that would match `f.js` & `o.js`, or am I mistaken?

**Comment by fisker** [comment]
I don't think we  need treat it as glob, `prettier [foo].js` we just check `[foo].js`.


So `prettier index*.js` only format file named exactly `index*.js`, won't match `index1.js` even only one file matched.
